check.go

```diff
@@ -124,14 +126,14 @@ func (c *CheckRunner) updateCheckHTTP(latestCheck *api.HealthCheck, checkHash ty
 			return false
 		}
 
-		c.logger.Printf("[INFO] Updating HTTP check %q", checkHash)
+		c.logger.Info("Updating HTTP check ", "checkHash", hclog.Fmt("%q", checkHash))
 
 		if httpCheckExists {
 			httpCheck.Stop()
 		} else {
 			tcpCheck, tcpCheckExists := c.checksTCP[checkHash]
 			if !tcpCheckExists {
-				c.logger.Printf("[WARN] Inconsistency check %q - is not TCP and HTTP", checkHash)
+				c.logger.Warn("Inconsistency check ", "checkHash", hclog.Fmt("%q - is not TCP and HTTP", checkHash))
```

Comment:
I think it would more similar to your other logs to have something like below:

```
c.logger.Warn("Inconsistency check is not TCP and HTTP", "checkHash", hclog.Fmt("%q", checkHash))
```